server/implementation-cdi/src/main/java/io/smallrye/graphql/cdi/validation/ValidationService.java

```diff
@@ -39,4 +40,13 @@ public void beforeInvoke(InvokeInfo invokeInfo) throws Exception {
     public String getConfigKey() {
         return ConfigKey.ENABLE_VALIDATION;
     }
+
+    private ValidatorFactory getValidatorFactory() {
+        try {
+            ValidatorFactory validatorFactory = CDI.current().select(ValidatorFactory.class).get();
+            return validatorFactory;
+        } catch (Throwable t) {
```

Comment:
```suggestion
        } catch (Exception ex) {
```

Also mentioned in the meeting, maybe catch Exception or RuntimeException instead of Throwable.